Sources/Target/ProcessBase.cpp

```diff
@@ -202,6 +203,7 @@ ErrorCode ProcessBase::afterResume() {
   // Disable breakpoints and try to hit the breakpoint.
   //
   disableBreakpointManager(softwareBreakpointManager());
+  disableBreakpointManager(hardwareBreakpointManager());
```

Comment:
I think `enableBreakpointManager` and `disableBreakpointManager` do not exist at this point. Looks like they're added in a later commit.